README.md

```diff
@@ -1 +1,48 @@
-# week1-helloMicroverse-project
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+ This project is written for purpose of learning and setup the first Microverse project.
+## Built With
+
+- HTML and CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is hello Microverse project.**
+**Built with simple index.html and style.css file.**
+
+
+## Authors
+Tesfaye Teklu
+👤 ****
+
+- GitHub: [@githubhandle](https://github.com/eyoab01)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As the last code reviewer said, lease 🙏🏽 follow the [README template](https://github.com/microverseinc/readme-template). All the headers should be present. Kindly add the LICENSE file and make sure that the license link works as expected.